charts/openebs/openebs-control-plane-pod-delete/engine.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: openebs-control-plane-chaos
+  namespace: openebs
+spec:
+  # It can be true/false
+  annotationCheck: 'false'
+  auxiliaryAppInfo: ''
```

Comment:
Can you please remove this field

Reply:
Why?

Reply:
We can see other application while deleting the control plane